src/Server/GraphiteFinder.cpp

```diff
@@ -0,0 +1,273 @@
+
```

Comment:
Code should be formatted in accordance with our [code style](https://clickhouse.com/docs/en/development/style/).  Most of the work could be done with `clang-format`.